front/src/UI/organisms/ChatPanel/index.tsx

```diff
@@ -58,43 +40,160 @@ export const ChatPanel = ({ ...props }) => {
     bg: 'gray.200',
   };
 
+  //props
+  const { chatListColumns, chatListType, chatListTabs, userID /*metadatas, setmetadata*/ } = props;
+
+  // react hooks
+  const [chatsTotal, setChatsTotal] = useState<number | undefined>();
+
+  // paginator variables
+  const { isDisabled, pagesQuantity, currentPage, setCurrentPage } = usePaginator({
+    total: chatsTotal,
+    initialState: {
+      pageSize: CHAT_PAGE_SIZE,
+      currentPage: CHAT_DEFAULT_PAGE,
+      isDisabled: false,
+    },
+  });
+
+  // query
+  const { loading, error, data, refetch } = useQuery(GET_CHATS, {
+    variables: { userID: userID, type: chatListTabs[0].type, page: CHAT_DEFAULT_PAGE },
+    nextFetchPolicy: 'network-only',
+  });
+
+  //mutation
+  const [updateChatToDead] = useMutation(UPDATE_CHAT, {
+    onCompleted: () => {
+      // 나의채팅방, 1:1채팅방 페이지를 첫 페이지로 변경
+      setCurrentPage(CHAT_DEFAULT_PAGE);
+      refetch({ page: CHAT_DEFAULT_PAGE });
+    },
+  });
+
+  const [createChat] = useMutation(CREATE_CHAT, {
+    onCompleted: () => {
+      refetch();
+    },
+  });
+
+  //채팅방 떠나기
+  const leaveChat = (uuid: string, ownerID: string, userID: string[]) => {
+    let leftChat = {}; // 나간 채팅방 정보가 담김.
+    if (ownerID === 'yshin') {
+      // TODO: yshin을 session 값으로 바꿔야 함.
+      leftChat = {
+        uuid: uuid,
+        isAlive: false,
+      };
+    } else {
+      leftChat = {
+        uuid: uuid,
+        userID: userID.filter((user) => user !== 'yshin'), //TODO: 'yshin'을 session 값으로 바꿔야 함.
+      };
+    }
+    updateChatToDead({
+      variables: {
+        newChat: leftChat,
+      },
+    });
+  };
+
+  //채팅방 생성
+  const createChatFunc = ({ name, type, password }: { name: string; type: 'public' | 'private'; password: string }) => {
+    let newChat = {};
+    if (type === 'public') {
+      newChat = { name: name, type: type, ownerID: 'yshin' }; //TODO: 'yshin' session 값으로 바꿔야 함.
+    } else {
+      //private
+      newChat = { name: name, type: type, password: password, ownerID: 'yshin' }; //TODO: 'yshin' session 값으로 바꿔야 함.
+    }
+    createChat({
+      variables: {
+        newChat: newChat,
+      },
+    });
+  };
+
+  //userID가 prop으로 들어오면(나의채팅방, 1:1채팅방) 버튼 생성
+  const createChatButton = !userID ? <CreateChat createChat={createChatFunc} /> : <></>;
```

Comment:
```suggestion
  const createChatButton = !userID ? <CreateChat createChat={createChatFunc} /> : null;
```

fragment는 DOM에 별도의 노드를 추가하지 않고 여러 자식을 그룹화하기 위해 사용하는데 위 사용 예시는 반환이 없는것을 기대해서 적절하지 않습니다.

https://ko.reactjs.org/docs/fragments.html
https://reactjs.org/docs/react-component.html#render